lib/anycable/rack/connection.rb

```diff
@@ -1,33 +1,35 @@
 # frozen_string_literal: true
 
-require 'anycable/rack-server/rpc/client'
-require 'anycable/rack-server/logging'
-require 'anycable/rack-server/errors'
+require "nanoid"
+require "set"
+require "json"
+
+require "anycable/rack/rpc/client"
+require "anycable/rack/logging"
+require "anycable/rack/errors"
 
 module AnyCable
-  # rubocop:disable Metrics/LineLength
-  module RackServer
-    class Connection
-      # rubocop:enable Metrics/LineLength
+  module Rack
+    class Connection # :nodoc:
       include Logging
 
       attr_reader :coder,
-                  :header_names,
+                  :headers,
                   :hub,
                   :socket,
                   :rpc_client,
-                  :server_id
+                  :sid
 
-      def initialize(socket, hub, coder, host, header_names, server_id)
-        @socket       = socket
-        @coder        = coder
-        @hub          = hub
-        @header_names = header_names
-        @server_id    = server_id
+      def initialize(socket, hub:, coder:, rpc_host:, headers:)
+        @socket = socket
+        @coder = coder
+        @headers = headers
+        @hub = hub
+        @sid = Nanoid.generate(size: 10)
```

Comment:
Why not `SecureRandom.hex(10)`?

Reply:
Don't know) probably, just for fun; it would be better to avoid unnecessary dependency